reactive/smtp_relay.py

```diff
@@ -0,0 +1,107 @@
+import os
```

Comment:
Can you include the new files in the [tox linting target](https://github.com/canonical/smtp-relay-operator/blob/main/tox.ini#L61)?